src/libs/actions/Report.ts

```diff
@@ -644,6 +686,13 @@ function openReport(
         parameters.groupChatAdminLogins = currentUserEmail;
         parameters.optimisticAccountIDList = participantAccountIDList.join(',');
         parameters.reportName = newReportObject.reportName ?? '';
+
+        // If we have an avatar then include it with the parameters
+        if (avatar) {
+            parameters.file = avatar;
+        }
+
+        clearGroupChat();
```

Comment:
This call is found to be done too early making the UI unexpectedly empty https://github.com/Expensify/App/issues/57021. FWIW This was not a bug at the time of writing, it must be some onyx change that made the onyx execution happen too fast before navigation.